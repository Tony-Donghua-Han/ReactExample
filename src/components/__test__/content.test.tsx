import React from "react";
import { shallow } from "enzyme";
import Content from "../content";

describe("Test content rendering", (): void => {
    test("Content should render without breaking", (): void => {
        const wrapper = shallow(<Content />);
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
