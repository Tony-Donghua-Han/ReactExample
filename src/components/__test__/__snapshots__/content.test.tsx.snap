// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test content rendering Content should render without breaking 1`] = `
"<div id=\\"content\\" className=\\"content\\">
  <div className=\\"absolute-centered center-text\\">
    Content of body
  </div>
</div>"
`;
